schemas/common/extensible.schema.json

```diff
@@ -50,7 +50,10 @@
             "^dsp:.*$": {},
             "^searchads:.*$": {},
             "^adcloud:.*$": {},
-            "^id3:.*$": {},
```

Comment:
Why did you delete `id3`???

Reply:
+1 - this should not be removed. This namespace is used.

Reply:
There was no intention to remove this. I added this back